radis/misc/config.py

```diff
@@ -355,28 +356,24 @@ def convertRadisToJSON():
     return
 
 
-def automatic_conversion():
+def check_and_convert():
     """Checks whether `~/radis.json` exists.
     If not then we use
     :func:`~radis.misc.config.convertRadisToJSON`
     to convert `~/.radis` into `~/radis.json`
     """
-
     # If `~/.radis` and `~/radis.json` both found, raises DeprecationWarning
     if exists(CONFIG_PATH_JSON) and exists(CONFIG_PATH):
         warnings.warn(
-            DeprecationWarning(
-                "`~/.radis` and `~/radis.json` both found. "
-                + "`~/.radis` config file was replaced by `~/radis.json` in version '0.9.29'."
-                + " Remove `~/.radis` to remove this warning.",
-            )
+            "`~/.radis` config file was replaced by `~/radis.json` in version '0.9.29'."
+            + " Remove `~/.radis` to remove this warning (make a copy to be sure).",
+            UserWarning,
```

Comment:
It's working,  I'm surprised that earlier it wasn't showing any warning message if both `.radis` and `radis.json` existed.

Reply:
somehow DeprecationWarning got suppressed on my setting.